crates/derive/src/stages/channel_reader.rs

```diff
@@ -99,6 +99,9 @@ impl<P> BatchQueueProvider for ChannelReader<P>
 where
     P: ChannelReaderProvider + OriginAdvancer + OriginProvider + ResettableStage + Send + Debug,
 {
+    fn flush(&mut self) { /* noop */
```

Comment:
Wonder if we should make a new set of traits here, since we plan on always including the `BatchStream`? This doesn't need to be here, but I'm cool w/ keeping the no-op.